provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -167,51 +167,54 @@ spec:
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: podifs.aci.aw
+  name: nodepodifs.aci.aw
```

Comment:
Since our interim plan is to keep podifs intact such that gbpserver keeps working, we should be adding nodepodifs, but not removing podifs.

Reply:
Updated the PR